stl/inc/ranges

```diff
@@ -1393,8 +1393,7 @@ namespace ranges {
 
             _NODISCARD friend constexpr difference_type operator-(const _Iterator& _Left,
                 const _Iterator& _Right) noexcept(noexcept(_Left._Current - _Right._Current)) /* strengthened */
-                requires sized_sentinel_for<iterator_t<_Base>, iterator_t<_Base>> { // constraints per LWG issue
-                                                                                    // unnumbered as of 2020-09-03

```

Comment:
For the record, this was LWG-3483 `transform_view::`*`iterator`*'s difference is overconstrained.